doc/content/the-things-stack/monitor/receivers-and-profiles/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Receiving alert notifications"
```

Comment:
This `receivers-and-profiles` folder should be under the `alerting` folder for the correct hierarchy.